README.md

```diff
@@ -57,95 +57,224 @@ Some of the skills/concepts this assignment aims to test (and build upon):
 * Routing (URL fragment based routing)
 
 ## API
-We can build API documentation using [Swagger](https://swagger.io/) or API Blueprint. 
+We can build API documentation using [Swagger](https://swagger.io/) or API Blueprint.
 A simple express JS API server can be built that is provided to students to allow them to develop offline. It could also be hosted somewhere to ease development when online?
 
 TODO: Someone will need to build a API server to support all of this client side functionality.
 
 The following specification is a WIP and only includes some endpoints.
 
+> POST `/login`
+
 ```json
-POST /login
 {
   "username": "string",
   "password": "string",
 }
+```
+
 returns:
+  - `200` OK for successful login, sets a cookie with a session token
+  - `403` for invalid credentials
+
+
+> POST `/signup`
 
-200 OK for successful login
-403 for invalid credentials
-POST /signup
+```json
 {
   "displayName": "string",
   "username": "string",
```

Comment:
email over displayName and username?

Reply:
username && displayName is overkill. What's the difference? Just more data. should be email, username, password

Reply:
I dunno dude you put displayName in the original spec i just left it there

Reply:
XD. @blame @DarkPurple141. I'm not always right, but defs remove duplication.